tests/textlab_api/textlab_api_test.py

```diff
@@ -13,3 +32,316 @@ def test_normalize_text(
     Test for the text normalization function that is applied before sending text to the TextLab API
     """
     assert output == normalize_text(input)
+
+
+def update_page_content(
+    login_role_user: tuple[Client, str],
+    title: str,
+    content: str,
+    hix_ignore: bool = False,
+) -> tuple[str, HttpResponse]:
+
+    # Log the user in
+    client, _role = login_role_user
+
+    # Update page content
+    edit_page = reverse(
+        "edit_page",
+        kwargs={
+            "region_slug": "augsburg",
+            "language_slug": "de",
+            "page_id": 2,
+        },
+    )
+    edit_page_data = {
+        "title": title,
+        "content": content,
+        "mirrored_page_region": "",
+        "_ref_node_id": 3,
+        "_position": "right",
+        "status": status.PUBLIC,
+    }
+    if hix_ignore:
+        edit_page_data["hix_ignore"] = "on"
+
+    return (edit_page, client.post(edit_page, edit_page_data))
```

Comment:
```suggestion
    return (edit_page, login_role_user[0].post(edit_page, edit_page_data))
```
Another nit-peaky and very optional comment:
Since we don't use _role variable here anyway, we may not create it.